code/modules/mob/living/carbon/human/species.dm

```diff
@@ -622,30 +625,33 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		var/obj/item/bodypart/r_leg/right_leg = H.get_bodypart(BODY_ZONE_R_LEG)
 		var/obj/item/bodypart/l_leg/left_leg = H.get_bodypart(BODY_ZONE_L_LEG)
 		var/datum/sprite_accessory/markings = GLOB.moth_markings_list[H.dna.features["moth_markings"]]
+		var/markings_icon_state = markings.icon_state
```

Comment:
We don't really need this var

Reply:
Tried it, didn't work. It showed the burned off markins after getting burned normally, but they wouldn't go away if the trait was removed. So I had to use my original method.

Reply:
This does still irk me, I'm going to look at it myself tomorrow

Reply:
Please do, I couldn't figure out why the other method wouldn't work.